src/QsFmt/Formatter/SyntaxTree/Node.fsi

```diff
@@ -85,6 +85,26 @@ type internal 'a Tuple =
         CloseParen: Terminal
     }
 
+/// A prefix operator. The operator is in the front of the operand.
+type internal 'a PrefixOperator =
+    {
+        /// The operator.
+        PrefixOperator: Terminal
+
+        /// The operand.
+        Operand: 'a
+    }
+
+/// A prefix operator. The operator is after the operand.
+type internal 'a PostfixOperator =
+    {
+        /// The operand.
+        Operand: 'a
+
+        /// The operator.
+        PostfixOperator: Terminal
```

Comment:
Actually, I used the field name `Operator` at first. However, it leads to errors because the compiler is confused between `PrefixOperator` and `PostfixOperator`. According to [F# grammar](https://fsharpforfunandprofit.com/posts/records/#label-order), the order of fields does **not** matter in a `Record`. Therefore, `{Operand: 'a, Operator: Terminal}` is the same as `{Operator: Terminal, Operand: 'a}`. To define both `PrefixOperator` and `PostfixOperator`, I have to differentiate a field name. Renaming the `Operator` field seems a convenient solution. Another solution would be to define `PrefixOperator` as `{Operator: Terminal, Right: 'a}`, which is consistent with `BinaryOperator`. But on itself, the name `Right` seems a little weird. 

I will change the `Operator` field of `BinaryOperator` to `InfixOperator` in a separate commit. So, if you prefer the `Operator`-`Right` solution, we can easily roll back and change.

Reply:
You can include the record name when constructing it, but it's sort of awkward, so I understand if you want to make the field names different.

```fsharp
{ PrefixOperator.Operator = foo; Operand = bar }
{ Operand = foo; PostfixOperator.Operator = bar }
```

I'm OK with using `Operand` for both. However, if you want to call the field `InfixOperator`, do you also want to rename `BinaryOperator` to `InfixOperator` for consistency?

Reply:
Yes, that would nice. I noticed that [FParsec](http://www.quanttec.com/fparsec/reference/operatorprecedenceparser.html) defines `InfixOperator`, `PrefixOperator`, and `PostfixOperator`, and they have been used in [QsExpressionParsing.fs](https://github.com/microsoft/qsharp-compiler/blob/main/src/QsCompiler/TextProcessor/QsExpressionParsing.fs#L87). So, maybe this group of names is a nice choice?

I have made this change in commit [b731e605](https://github.com/microsoft/qsharp-compiler/pull/1068/commits/b731e605fa66a92606a74291b318b307e0c27f38). See if you like it. :)

Reply:
Looks good. :)